README.md

```diff
@@ -185,13 +212,22 @@ Try install [Microsoft Visual C++ Redistributable](https://aka.ms/vs/16/release/
 
 ## Build ZENO
 
-- Linux
+- Linux(Previous)
 
 ```bash
 cmake -B build
 make -C build -j8
 ```
 
+- Ubuntu 20.04
+
+```bash
+# clone ZENO
+git clone git@github.com:zenustech/zeno.git
+cd zeno
+make all -j # you can change cmake configuration using $ ccmake -Bbuild
```

Comment:
what's the difference?

Reply:
Sloving, add optional ccmake option to change cmake configurations.